src/js/util.js

```diff
@@ -1464,7 +1464,12 @@ export function isValidationErrorChanged (currErr, prevErr) {
   if (prevErr.length !== currErr.length) { return true }
 
   for (let i = 0; i < currErr.length; ++i) {
-    const pErr = prevErr.find(p => p.dataPath === currErr[i].dataPath && p.schemaPath === currErr[i].schemaPath)
+    let pErr
```

Comment:
No big deal, but in general I try to use `const` as much as possible. Often it's possible to write a conditional like:

```js
const pErr = (currErr[i].type === 'error')
  ? prevErr.find(p => p.line === currErr[i].line)
  : prevErr.find(p => p.dataPath === currErr[i].dataPath && p.schemaPath === currErr[i].schemaPath)
```